IT-Academy-Social-Projects-KRV/CMIYC
# Feature 112 code coverage plugin lera

## How to test plugin:
1. Run Auth  App (for integration tests)
2. In terminal: PS [your_pass]\CMIYC\Backend\microservices\search> `mvn clean test` 
3. Go to the target/site/jacoco/index.html
4. You should see:
![image](https://user-images.githubusercontent.com/75213424/165797343-60bd410e-d432-40f1-8cd7-76984cc9b8f3.png)

## Erors:
- If says that mvn command not exists - install maven 3.8.5 (https://phoenixnap.com/kb/install-maven-windows).
- If says that JAVA_HOME not exists - install java 11 (https://alexnettm.com/ru/windows/kak-ustanovit-java-v-windows-11.html)
### Your enviroment variables should look like:
![image](https://user-images.githubusercontent.com/75213424/165799051-3c228934-c0b9-479a-810c-4e0912cdbeb7.png)
![image](https://user-images.githubusercontent.com/75213424/165798876-09f55107-17be-42e3-b905-1db9f8dabcb9.png)
